helper/src/components/addonsTab.js

```diff
@@ -422,6 +422,22 @@ export default function ({ tabValues, updateFn, featureFlag, invalidArray }) {
                     label="Enable high availability mode"
                 />
             </Stack.Item>
+
+            <Separator className="notopmargin" />
+
+            <Stack.Item align="start">
+                <Label required={true}>
+                    Azure Blob storage CSI driver: Enable BlobFuse or NFS v3 access to Azure Blob
```

Comment:
Can you rename the label to be `CSI Blob storage: Enable BlobFuse or NFS v3 access to Azure Blob Storage`